src/modules/aws_rds/mappers/db_cluster.ts

```diff
@@ -0,0 +1,346 @@
+import {
+  RDS,
+  DBCluster as AWSDBCluster,
+  CreateDBClusterCommandInput,
+  waitUntilDBClusterAvailable,
+  paginateDescribeDBClusters,
+  ModifyDBClusterCommandInput,
+  DeleteDBClusterMessage,
+  waitUntilDBClusterDeleted,
+  DescribeDBClustersCommandInput,
+} from '@aws-sdk/client-rds';
+import { createWaiter, WaiterOptions, WaiterState } from '@aws-sdk/util-waiter';
+
+import { AwsRdsModule } from '..';
+import { AWS, crudBuilderFormat, paginateBuilder } from '../../../services/aws_macros';
+import { awsSecurityGroupModule } from '../../aws_security_group';
+import { Context, Crud, MapperBase } from '../../interfaces';
+import { DBCluster, dbClusterEngineEnum } from '../entity/db_cluster';
+
+export class DBClusterMapper extends MapperBase<DBCluster> {
+  module: AwsRdsModule;
+  entity = DBCluster;
+  equals = (a: DBCluster, b: DBCluster) =>
+    Object.is(a.allocatedStorage, b.allocatedStorage) &&
+    Object.is(a.iops, b.iops) &&
+    Object.is(a.backupRetentionPeriod, b.backupRetentionPeriod) &&
+    Object.is(a.dbClusterInstanceClass, b.dbClusterInstanceClass) &&
+    Object.is(a.deletionProtection, b.deletionProtection) &&
+    Object.is(a.engine, b.engine) &&
+    Object.is(a.engineVersion, b.engineVersion) &&
+    !a.masterUserPassword && // Special case, if master password defined, will update the password
```

Comment:
Note for the future: I know we do this in normal RDS instances, too, but with RPCs, we may want to pull this out and have an `update_cluster_password` RPC function here. Not blocking.

Reply:
let me create an issue for it